src/tribler-gui/tribler_gui/widgets/tablecontentdelegate.py

```diff
@@ -187,26 +187,6 @@ def draw_channel_state(self, painter, option, index, data_item):
         return True
 
 
-class SubscribedControlMixin(object):
```

Comment:
Are the `SubscribedControlMixin` and `SubscribeToggleControl` unused?

Reply:
Yes! I found this unused code by running the GUI against the real Core. Though, I'll probably readd it shortly, better to delete it now as it is really unused (voting column does the subscription management).

Reply:
Ah good! Yeah I'm in favour of removing any dead code, even it is re-added later on. We can always fetch it from the VCS histories.